.github/workflows/cli-scripts-deploy-managed-online-endpoint-access-resource-uai.yml

```diff
@@ -10,6 +10,8 @@ on:
       - cli/deploy-managed-online-endpoint-access-resource-uai.sh
       - .github/workflows/cli-scripts-deploy-managed-online-endpoint-access-resource-uai.yml
       - cli/setup.sh
+      - cli/endpoints/online/managed/managed-identity
```

Comment:
did you mean this folder instead?: cli/endpoints/online/managed/managed-identities